optional_plugins/html/avocado_result_html/templates/results.html

```diff
@@ -50,8 +50,9 @@ <h4 class="panel-title">Summary</h4>
             <tr>
               <td>Stats</td>
               <td>From {{ data.result.tests_total }} tests executed,
-                {{ data.result.passed }} passed and
-                {{ data.result.warned }} warned - success rate of
+                <span class="pass">{{ data.result.passed }} passed</span>,
+                <span class="fail">{{ data.result.failed }} failed</span>, and
```

Comment:
Adding total failed here, might confuse users, because success rate is computed as `(FAIL+ERROR+INTERRUPT+PASS+WARN)/(PASS+WARN)`. If you omit `ERROR` and `INTERUPT` you can get sentence like this:
```
From 4 tests executed, 1 passed, 1 failed, and 1 warned - success rate of 50.00% (excluding SKIP and CANCEL)
```
Which is IMO confusing.

Reply:
I add ERROR + INTERRUPTED to the calculation. Or would you prefer to to drop the "did not pass" output entirely?